tests/integration/helpers.py

```diff
@@ -546,7 +546,20 @@ async def dispatch_workflow(
     """
     start_time = datetime.now(timezone.utc)
 
+    # TODO: Debug
+    print("##################################################################")
```

Comment:
Would logging module be helpful here? Since I'm not sure if print gets captured in the log output.